### ISSUE TITLE
Teleport command crash

### ISSUE THREAD

**Issue by fvnkhead** [x3Karma/Admin-Abuse-Mod#3]
My server crashed with this error when I tried to teleport with this command: `teleport ErliteDev`. Does the command break with only one argument?

```
[16:57:11] [info] [SERVER SCRIPT] SCRIPT ERROR: [SERVER] Index "1" is beyond array size of 1
[16:57:11] [info] [SERVER SCRIPT]  -> switch (args[1])
[16:57:11] [info] [SERVER SCRIPT]
CALLSTACK
*FUNCTION [TeleportCMD()] teleport.nut line [89]
*FUNCTION [CodeCallback_ClientCommand()] _mapspawn.gnut line [209]

[16:57:11] [info] [SERVER SCRIPT] LOCALS
[useCrosshair] false
[sheep2] NULL
[sheep1] ARRAY
[players] ARRAY
[weaponId] ""
[weapon] NULL
[args] ARRAY
[player] ENTITY (player fvnkhead [1] (player "fvnkhead" at <920.573 -816.756 1.03125>))
[this] TABLE
[commandString] "teleport"
[args] ARRAY
[player] ENTITY (player fvnkhead [1] (player "fvnkhead" at <920.573 -816.756 1.03125>))
[this] TABLE

DIAGPRINTS

[16:57:11] [info] [SERVER SPEW_WARNING] Script function CodeCallback_ClientCommand() must return a bool.
0024:fixme:msvcp:_Locinfo__Locinfo_ctor_cat_cstr (00000000007CE320 1 C) semi-stub
0024:fixme:msvcp:_Locinfo__Locinfo_ctor_cat_cstr (00000000007CE070 1 C) semi-stub
[16:57:11] [error] Northstar has crashed! a minidump has been written and exception info is available below:
[16:57:11] [error] Cause: Access Violation
Attempted to read from: 0x00000000
[16:57:11] [error] At: engine.dll + 0x247d3a
[16:57:11] [error]     Northstar.dll + 0x128ea47a
[16:57:11] [error]     ntdll.dll + 0x17002567d
[16:57:11] [error]     ntdll.dll + 0x1700548c5
[16:57:11] [error]     ntdll.dll + 0x17005245e
[16:57:11] [error]     engine.dll + 0x18677d3a
[16:57:11] [error]     engine.dll + 0x18847466
[16:57:11] [error]     engine.dll + 0x18550c2f
[16:57:11] [error]     engine.dll + 0x18550542
[16:57:11] [error]     engine.dll + 0x1859db55
[16:57:11] [error]     engine.dll + 0x185f88c0
[16:57:11] [error]     Northstar.dll + 0x128d96d6
[16:57:11] [error]     engine.dll + 0x185f652f
[16:57:11] [error]     engine.dll + 0x187c0e2d
[16:57:11] [error]     engine.dll + 0x185f79d9
[16:57:11] [error]     engine.dll + 0x185f793c
[16:57:11] [error]     launcher.dll + 0x12b05afd
[16:57:11] [error]     launcher.dll + 0x12b05afd
[16:57:11] [error]     launcher.dll + 0x12afd386
[16:57:11] [error]     NorthstarLauncher.exe + 0x140003e7d
[16:57:11] [error]     NorthstarLauncher.exe + 0x140007000
[16:57:11] [error]     kernel32.dll + 0x7b62c599
[16:57:11] [error]     ntdll.dll + 0x170059d53
[16:57:11] [error]     NorthstarLauncher.exe + 0x0
[16:57:11] [error]     NorthstarLauncher.exe + 0x140007070
[16:57:11] [error]     NorthstarLauncher.exe + 0x67ff0000
[16:57:11] [error]     NorthstarLauncher.exe + 0x0
[16:57:11] [error] RAX: 0x0
[16:57:11] [error] RBX: 0x18c156d0
[16:57:11] [error] RCX: 0x0
[16:57:11] [error] RDX: 0x0
[16:57:11] [error] RSI: 0x2b50bb78
[16:57:11] [error] RDI: 0x2b50bb78
[16:57:11] [error] RBP: 0x2b50bd88
[16:57:11] [error] RSP: 0x7cf470
[16:57:11] [error] R8: 0x2b50bb78
[16:57:11] [error] R9: 0x65
[16:57:11] [error] R10: 0x2b50bd94
[16:57:11] [error] R11: 0x18a378ec
[16:57:11] [error] R12: 0x0
[16:57:11] [error] R13: 0x0
[16:57:11] [error] R14: 0xffffffff
[16:57:11] [error] R15: 0x1
0024:fixme:dbghelp_dwarf:dwarf2_read_range no entry found
0024:fixme:dbghelp_dwarf:dwarf2_read_range no entry found
0024:fixme:dbghelp:MiniDumpWriteDump NIY MiniDumpScanMemory
```

**Comment by x3Karma** [comment]
apparently it does lmao

**Comment by x3Karma** [comment]
Should be fixed in v1.2.3
